project/src/components/filter-item/filter-item.tsx

```diff
@@ -0,0 +1,26 @@
+import './style.css';
+import {useDispatch, useSelector} from 'react-redux';
+import {changeGenre} from '../../store/action';
+
+type FilterItemProps = {
+  genre: string;
+}
+
+function FilterItem(props: FilterItemProps): JSX.Element {
+  const {genre} = props;
+  const activeFilter = useSelector((state: { genre: string; }) => state.genre);
```

Comment:
Такая запись типов приведёт к большому дублированию. И если стор поменяется, то нужно будет менять везде. А если забыть поменять, это приведёт к ошибке. Поэтому лучше завести отдельный тип стора и использовать везде его.
```suggestion
  const activeFilter = useSelector((state: StoreState) => state.genre);
```